services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/team/MembershipInvitationManagerImplTest.java

```diff
@@ -261,6 +263,17 @@ public void testGetOpenSubmissionsForTeamAndRequesterInRangeUnauthorized() throw
 	@Test
 	public void testCreateInvitationNotification() throws Exception {
 		MembershipInvtnSubmission mis = createMembershipInvtnSubmission(MIS_ID);
+		testCreateInvitationNotificationHelper(mis);
+	}
+	
+	@Test
+	public void testCreateInvitationNotificationWithNullCreatedOn() throws Exception {
+		MembershipInvtnSubmission mis = createMembershipInvtnSubmission(MIS_ID);
+		mis.setCreatedOn(null);
+		testCreateInvitationNotificationHelper(mis);
```

Comment:
testCreateInvitationNotification() is fully testing that membershipInvitationManagerImpl.createInvitationNotification() returns the correct values with createdOn field being set in mis. 

This test, testCreateInvitationNotificationWithNullCreatedOn(), should only test that the method doesn't fail for mis with createdOn set to null. The behavior we expect is that the method doesn't fail when createdOn is null. We should not expect that the link still be the same. This will make the test unstable when it's run on a slower machine.